packages/headless/src/utils/errors.ts

```diff
@@ -1 +1,9 @@
 export const loadReducerError = new Error('Failed to load reducers.');
+
+export class ExpiredTokenError extends Error {
+  constructor() {
+    super();
+    this.name = 'ExpiredToken';
+    this.message = 'The token being used to perform the request is expired.';
```

Comment:
Should there be any information in that message that explains the `renewAccessToken` configuration option exists ?

I guess it could also be worth it to have some article about this docs.coveo.com/headless

Reply:
I added a mention to the option inside the middleware since it is responsible for invoking the function. I think it is cleaner to mention it there than in the thrown error.

An article about "tokens and headless" would be a good idea. How to generate them, how to renew them etc.
@dlutzCoveo what do you think?